clinica/utils/freesurfer.py

```diff
@@ -146,61 +56,217 @@ def generate_regional_measures_alt(
             "normMax",
             "normRange",
         ]
+    else:
+        raise ValueError(
+            f"Unknown column type {column_type}. Use either parcellation or segmentation."
+        )
+
+    if not stats_filename.exists():
+        raise FileNotFoundError(f"Stats file {stats_filename} could not be found.")
+
+    return pd.read_csv(
+        stats_filename,
+        names=columns,
+        comment="#",
+        header=None,
+        delimiter="\s+",
+        dtype=str,
+    )
```

Comment:
A bit more Pythonic:

```suggestion
    try:
        return pd.read_csv(
            stats_filename,
            names=columns,
            comment="#",
            header=None,
            delimiter="\s+",
            dtype=str,
        )
    except FileNotFoundError:
        raise FileNotFoundError(f"Stats file {stats_filename} could not be found.")
```